sxg_rs/src/acme/directory.rs

```diff
@@ -0,0 +1,135 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! This module includes the URL, request and response of the ACME operations.
+
+use crate::fetcher::Fetcher;
+use anyhow::{Error, Result};
+use chrono::offset::FixedOffset;
+use chrono::DateTime;
+use serde::{Deserialize, Serialize};
+
+pub const LETSENCRYPT_STAGING: &str = "https://acme-staging-v02.api.letsencrypt.org/directory";
```

Comment:
Probably best to get this from a config file and not hard code it.

Reply:
Also note that LetsEncrypt ACME doesn't support SXGs.

Reply:
The hard-coded URL is now removed.